buckets/redis/bucket.go

```diff
@@ -67,6 +67,19 @@ func (bf *bucketFactory) Init(cfg *pbconfig.ServiceConfig) {
 	if bf.client == nil {
 		bf.connectToRedisLocked()
 	}
+
+	if err := bf.cleanStaleBuckets() ; err != nil {
```

Comment:
Could we do this in a goroutine? It shouldn't affect normal usage patterns, though we may want to make sure two updates can't overlap.